velox/connectors/tpch/CMakeLists.txt

```diff
@@ -12,7 +12,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-velox_add_library(velox_tpch_connector OBJECT TpchConnector.cpp)
+velox_add_library(velox_tpch_config OBJECT TpchConfig.cpp)
```

Comment:
Library velox_tpch_config is not required. Just adding TpchConfig to velox_tpch_connector is sufficient. Please delete lines 15 and 16.